src/types/line.js

```diff
@@ -85,9 +85,9 @@ export default class LineAnnotation extends Element {
       y >= labelRect.y - h2 && y <= labelRect.y + h2;
   }
 
-  inRange(x, y) {
+  inRange(mouseX, mouseY, useFinalPosition) {
     const epsilon = this.options.borderWidth || 1;
-    return this.intersects(x, y, epsilon) || this.isOnLabel(x, y);
+    return this.intersects(mouseX, mouseY, epsilon, useFinalPosition) || this.isOnLabel(mouseX, mouseY);
```

Comment:
isOnLabel uses current coordinates recardless of `useFinalPosition`

Reply:
should be fixed